mllib/src/test/scala/org/apache/spark/mllib/linalg/UDTSerializationBenchmark.scala

```diff
@@ -18,52 +18,52 @@
 package org.apache.spark.mllib.linalg
 
 import org.apache.spark.sql.catalyst.encoders.ExpressionEncoder
-import org.apache.spark.util.Benchmark
+import org.apache.spark.util.{Benchmark, BenchmarkBase => FileBenchmarkBase}
 
 /**
  * Serialization benchmark for VectorUDT.
+ * To run this benchmark:
+ * 1. without sbt: bin/spark-submit --class <this class> <spark mllib test jar>
```

Comment:
I think `<` should replaced to `[`:
```scala
[error] /home/jenkins/workspace/SparkPullRequestBuilder@2/sql/core/target/java/org/apache/spark/sql/DatasetBenchmark.java:5: error: unknown tag: this
[error]  * 1. without sbt: bin/spark-submit --class <this class> <spark sql test jar>
[error]                                             ^
[error] /home/jenkins/workspace/SparkPullRequestBuilder@2/sql/core/target/java/org/apache/spark/sql/DatasetBenchmark.java:5: error: unknown tag: spark
[error]  * 1. without sbt: bin/spark-submit --class <this class> <spark sql test jar>
[error]                                                          ^
[error] /home/jenkins/workspace/SparkPullRequestBuilder@2/sql/core/target/java/org/apache/spark/sql/DatasetBenchmark.java:6: error: unknown tag: this
[error]  * 2. build/sbt "sql/test:runMain <this class>"
[error]                                   ^
[error] /home/jenkins/workspace/SparkPullRequestBuilder@2/sql/core/target/java/org/apache/spark/sql/DatasetBenchmark.java:7: error: unknown tag: this
[error]  * 3. generate result: SPARK_GENERATE_BENCHMARK_FILES=1 build/sbt "sql/test:runMain <this class>"
[error]   
```

Reply:
+1 for fix the docs to pass Jenkins.
Also, could you rebase this PR to resolve conflicts, @seancxmao ?

Reply:
Thank you, @seancxmao . Happy festival! :)